src/components/products-list.vue

```diff
@@ -1,26 +1,42 @@
 
 <template>
-  <div :class="'product' + (index + 1)" v-for="(product, index) in products">
-    <img
-      :src="product.img"
-      imgfield="img"
-      style="max-width: 150px; width: 150px"
-      alt=""
-    />
+  <div :class="'product'" v-for="(product, index) in products">
+    <img :src="product.img" imgfield="img" style="max-width: 150px; width: 150px" alt="" />
     <p>{{ product.name }}</p>
+    <div class="product-item">
+      <img src="https://cdn.icon-icons.com/icons2/2785/PNG/512/trolley_cart_add_icon_177406.png" alt="Добавить в корзину"
+        width="50" height="50" @click="addToCart(product)">
+
+     <img src="https://cdn.icon-icons.com/icons2/2785/PNG/512/trolley_cart_success_icon_177398.png" alt="Уже в корзине"
```

Comment:
icons should be should in nature way. Skip delete from cart when it's not in cart. check docs https://vuejs.org/api/built-in-directives.html#v-else

create computed variables for example

```
const inCart = computed(()=>...) 
```